app/controllers/admin_routes/adminManagement.py

```diff
@@ -47,35 +47,39 @@ def adminSearch():
 
 @admin.route("/adminManagement/userInsert", methods=['POST'])
 def manageLaborAdmin():
-    if request.form.get("addAdmin"):
-        newAdmin = getUser('addAdmin')
-        addAdmin(newAdmin, 'labor')
-        adminFlashMessage(newAdmin, 'added', 'Labor')
-
-    elif request.form.get("removeAdmin"):
-        oldAdmin = getUser('removeAdmin')
-        removeAdmin(oldAdmin, 'labor')
-        adminFlashMessage(oldAdmin, 'removed', 'Labor')
-
-    elif request.form.get("addFinancialAidAdmin"):
-        newAdmin = getUser('addFinancialAidAdmin')
-        addAdmin(newAdmin, 'finAid')
-        adminFlashMessage(newAdmin, 'added', 'Financial Aid')
-
-    elif request.form.get("removeFinancialAidAdmin"):
-        oldAdmin = getUser('removeFinancialAidAdmin')
-        removeAdmin(oldAdmin, 'finAid')
-        adminFlashMessage(oldAdmin, 'removed', 'Financial Aid')
+    actionMap = {
+    "addAdmin":                {"type": "labor", "action": "add", "pretty": "Labor"},
+    "removeAdmin":             {"type": "labor", "action": "remove", "pretty": "Labor"},
+    "addFinancialAidAdmin":    {"type": "finAid", "action": "add", "pretty": "Financial Aid"},
+    "removeFinancialAidAdmin": {"type": "finAid", "action": "remove", "pretty": "Financial Aid"},
+    "addSAASAdmin":            {"type": "saas", "action": "add", "pretty": "SAAS"},
+    "removeSAASAdmin":         {"type": "saas", "action": "remove", "pretty": "SAAS"},
+    }
+    for key, meta in actionMap.items():
```

Comment:
You don't need to loop through the dictionary if you only have one key. You can just select the values from the dictionary directly:
`meta = actionMap.get(request.form.get(key))`